include/RevCPU.h

```diff
@@ -118,6 +118,7 @@ class RevCPU : public SST::Component{
     {"RDMAPerCycle",    "Number of RDMA messages per cycle to inject",  "1"},
     {"testIters",       "Number of PAN test messages to send",          "255"},
     {"splash",          "Display the splash logo",                      "0"},
+    {"disableCoprocClock",  "Disables the manual clocking of coprocessor subcomponents", "0"},
     )
```

Comment:
Can we call this "IndependentCoProcClock" or something similar? This sounds like the CoProc receives no clock at all

Reply:
changed in 9d29794